docs/reference/module-api/module-overview.md

```diff
@@ -14,6 +17,13 @@ Module configuration objects may use the following configuration properties.
 | [`filters`](#filters) | Object | Yes | Configure piece type filters | Piece |
 | [`columns`](#columns) | Object | Yes | Configure piece type manager columns | Piece |
 
+
+### "Cascading" settings
+
+Many Apostrophe module sections are structured as objects with `add`, `remove`, and sometimes `group` properties. This pattern allows these settings to "cascade" from the base classes through to project level classes. The subsections help Apostrophe manage inherited settings without requiring those inherited settings be re-declared by developers.
```

Comment:
```suggestion
Many Apostrophe module sections are structured as objects with `add`, `remove`, and sometimes `group` properties. This pattern allows these settings to "cascade" from the base classes to project level classes without requiring those settings to be redeclared.
```

Very nitpicky, sorry last one I swear.

Related question, "Many Apostrophe module sections". We don't say which ones **do** or **dont** have these sections. Is that at all relevant?